Firestore/Source/Remote/FSTSerializerBeta.mm

```diff
@@ -668,16 +672,19 @@ - (GCFSArrayValue *)encodedArrayTransformElements:(const std::vector<FSTFieldVal
 
 - (FSTMutationResult *)decodedMutationResult:(GCFSWriteResult *)mutation {
   // NOTE: Deletes don't have an updateTime.
-  FSTSnapshotVersion *_Nullable version =
-      mutation.updateTime ? [self decodedVersion:mutation.updateTime] : nil;
+  absl::optional<SnapshotVersion> version = absl::nullopt;
```

Comment:
Isn't the default constructor for `absl::optional` equivalent to this?

Reply:
done